src/plugins/exception-helpers.ts

```diff
@@ -11,32 +14,32 @@ export default function exceptionHelpersPlugin (app) {
   app.addHelper('websocketException', websocketException);
 }
 
-function exception (ctx, error) {
+function exception (ctx: MojoContext, error: Error): boolean {
   if (ctx.isWebSocket) return ctx.websocketException(error);
   return ctx.httpException(error);
 }
 
-async function htmlException (ctx, error) {
-  ctx.stash.exception = error;
+async function htmlException (ctx: MojoContext, error: Error): Promise<boolean> {
+  (ctx.stash as MojoStash).exception = error;
 
-  const mode = ctx.app.mode;
-  if (await ctx.render({view: `exception.${mode}`, maybe: true, status: 500}) === true) return;
-  if (await ctx.render({view: 'exception', maybe: true, status: 500}) === true) return;
+  const mode: string = ctx.app.mode;
+  if (await ctx.render({view: `exception.${mode}`, maybe: true, status: 500}) === true) return true;
+  if (await ctx.render({view: 'exception', maybe: true, status: 500}) === true) return true;
 
   const view = mode === 'development' ? 'mojo/debug' : 'mojo/exception';
   return ctx.render({view, status: 500});
 }
 
-async function htmlNotFound (ctx) {
-  const mode = ctx.app.mode;
-  if (await ctx.render({view: `not-found.${mode}`, maybe: true, status: 404}) === true) return;
-  if (await ctx.render({view: 'not-found', maybe: true, status: 404}) === true) return;
+async function htmlNotFound (ctx: MojoContext): Promise<boolean> {
+  const mode: string = ctx.app.mode;
+  if (await ctx.render({view: `not-found.${mode}`, maybe: true, status: 404}) === true) return true;
+  if (await ctx.render({view: 'not-found', maybe: true, status: 404}) === true) return true;
 
   const view = mode === 'development' ? 'mojo/debug' : 'mojo/not-found';
   return ctx.render({view, status: 404});
 }
 
-function httpException (ctx, error) {
+function httpException (ctx: MojoContext, error: Error): boolean {
```

Comment:
Those are not `boolean`, rendering always results in a `Promise`.

Reply:
Doesn't httpException have to be async as well then?